__tests__/database/schema/ChannelListSchema.test.ts

```diff
@@ -552,12 +553,13 @@ describe('TESTING ChannelListSchema', () => {
         rawJson: expect.any(Object),
         user: null,
         members: null,
-        channelPicture: '',
+        channelPicture:
+          'https://res.cloudinary.com/hpjivutj2/image/upload/v1680929851/default-profile-picture_vrmmdn.png',
         expiredAt: null
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `channelPicture` now has a default value which is a good practice. However, hardcoding URLs directly in the code can lead to maintenance issues if the URL changes in the future. Consider storing such URLs in a constants file or environment variables and referencing them here.
```diff
-        channelPicture: 'https://res.cloudinary.com/hpjivutj2/image/upload/v1680929851/default-profile-picture_vrmmdn.png',
+        channelPicture: DEFAULT_PROFILE_PICTURE_URL,
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->